nautobot/extras/tests/test_forms.py

```diff
@@ -1086,3 +1089,27 @@ def test_update_job_with_approval_required_and_has_has_sensitive_variables_is_tr
             error_msg["approval_required"][0]["message"],
             "A job that may have sensitive variables cannot be marked as requiring approval",
         )
+
+
+class ConfigContextFormTestCase(TestCase):
+    @override_settings(CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED=False)
+    def test_without_dynamic_groups(self):
+        form = ConfigContextForm()
+        self.assertIsNone(form.fields.get("dynamic_groups", None))
+
+    @override_settings(CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED=True)
+    def test_with_dynamic_groups(self):
+        form = ConfigContextForm()
+        self.assertIsNotNone(form.fields.get("dynamic_groups", None))
+
+
+class ConfigContextFilterFormTestCase(TestCase):
+    @override_settings(CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED=False)
+    def test_without_dynamic_groups(self):
+        filter = ConfigContextFilterForm()
+        self.assertIsNone(filter.fields.get("dynamic_groups", None))
+
+    @override_settings(CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED=True)
+    def test_with_dynamic_groups(self):
+        filter = ConfigContextFilterForm()
+        self.assertIsNone(filter.fields.get("dynamic_gorups", None))
```

Comment:
Please add docstrings to each of the `test_` methods so that they show in verbose mode. Thanks!